tests/integration/api/image_apis_test.go

```diff
@@ -1,4 +1,4 @@
-package api
+package api_test
```

Comment:
Why do we change the package name

Reply:
prefer to following the best practice from https://onsi.github.io/ginkgo/
> Go allows us to specify the books_test package alongside the books package. Using books_test instead of books allows us to respect the encapsulation of the books package: your tests will need to import books and access it from the outside, like any other package. This is preferred to reaching into the package and testing its internals and leads to more behavioral tests. You can, of course, opt-out of this – just change package books_test to package books